src/components/Navigator/QuickNavigationModal.vue

```diff
@@ -172,8 +172,12 @@ export default {
         processedInputRegex: new RegExp(constructFuzzyRegex(processedUserInput), 'i'),
         childrenMap,
       });
-      // Return the first 20 symbols out of sorted ones
-      return orderSymbolsByPriority(matches).slice(0, 20);
+      // Filter symbols with repeated paths and return the first 20
+      return orderSymbolsByPriority(
+        matches.filter((s1, index) => (
```

Comment:
Instead of filtering inside the call of `orderSymbolsByPriority` I suggest creating an `uniqueMatches` const outside and then pass it to `orderSymbolsByPriority`.

The comment of `// Filter symbols with repeated paths and return the first 20` should be on top of the new const instead of `orderSymbolsByPriority`

Reply:
Resolved https://github.com/apple/swift-docc-render/pull/496/commits/aa5168129b19e9b4f1643cf99059659b4fb42247